File: CodeIQ-q969/CodeIQ-q969-1.go
for i:=1;i<51;i++{if i%15<1{fmt.Printf("FIZZBUZZ")}else if i%3<1{fmt.Printf("fizz")}else if i%5<1{fmt.Printf("fizz")}else{fmt.Printf("%d",i)}}
s:="";for i:=1;i<51;i++{if i%15<1{s+="FIZZBUZZ"}else if i%3<1{s+="fizz"}else if i%5<1{s+="fizz"}else{s+=strconv.Itoa(i)}}fmt.Printf("%s",s)
for i:=1;i<51;i++{if i%15<1{fmt.Printf("FIZZBUZZ")}else if i%3<1{fmt.Printf("fizz")}else if i%5<1{fmt.Printf("buzz")}else{fmt.Printf("%d",i)}}

File: CodeIQ-q969/CodeIQ-q969-1.py
s=""
i=1
while i<51:
 if i%15<1:
  s+="FIZZBUZZ"
 elif i%3<1:
  s+="fizz"
 elif i%5<1:
  s+="buzz"
 else:
  s+=str(i)
 i=i+1
print s